### PR TITLE
Added GuideStar Seal of Transparency to the Footer

### PR DESCRIPTION
**Pull request by Akshansh99**
#150

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -17,6 +18,16 @@ export default function Footer() {
           width="212"
         />
       </a>
+      <Link href="https://www.guidestar.org/profile/84-4323038" target="_blank">
+        <a>
```
Comment (RedFox0x20) [inline components/Footer.js:22]: I believe this `<a>` tag is unnecessary and actually gets removed entirely during the rendering process.

**Review by RedFox0x20 — COMMENTED** [review]
The changes within the files here look fine, no problems at all with it. Thanks for the contribution!

In regards to the placement, I wonder if it would look better (for the footer as a whole) to have this inline with the `Vercel` image.

File: components/Footer.js (hunk 1 above)
Comment (Akshansh99) [reply components/Footer.js]: Hey I've put both `vercel` and Guidestar seal in inline position. Also, I've removed redundant `<a>`  tag.

**Review by RedFox0x20 — APPROVED** [review]
Thanks for the changes! Only thing that isn't there now is the cursor being set to `pointer`, it's fine without it however it helps to provide visual feedback that the image is clickable. Seems the `<a>` tag helped to add that functionality however it should be possible without the addition of that with `className="cursor-pointer"` on a wrapping span or even on the parent div. Strange that the `Link` component doesn't do this by default.

File: components/Footer.js [hunk 2]
```diff
@@ -9,14 +10,31 @@ export default function Footer() {
...
+          href="https://vercel.com?utm_source=margaritahumanitarian&utm_campaign=oss"
+        >
+          <Image
+            alt="Powered by Vercel"
+            height="44"
+            src="/images/powered-by-vercel.svg"
+            width="212"
+          />
+        </a>
+        <Link
```
Comment (marekrozmus) [inline components/Footer.js:25]: why do we use here `Link` if the above link for Vercel works OK with `<a>` tag? It looks like it causes more troubles than profits (do we have any :)) ? The `target="_blank"` seems to not work for `Link` component.

**Review by marekrozmus — CHANGES_REQUESTED** [review]

File: components/Footer.js (hunk 2 above)
Comment (Akshansh99) [reply components/Footer.js]: I've removed the `Link` and it runs fine on my local machine. It isn't deploying here though.

**Comment by RedFox0x20** [comment]
The error caused is as `Link` is still being imported:
```
19:17:04.887 | 2:8  Error: 'Link' is defined but never used.  no-unused-vars
```

Just needs removing if it's not being used. In this instance, Link provides no benefit, it's use case of for linking between pages within the same site, external links are often best left as `<a>` tags.

**Review by RedFox0x20 — APPROVED** [review]

**Comment by audreyfeldroy** [comment]
Thanks @Akshansh99 for the PR and @RedFox0x20 for the review, much appreciated! Merged.

**Pull request merged**